docker-compose.yml

```diff
@@ -71,16 +71,15 @@ services:
             ip link set vni10 up && \
             ip link set br10 up && \
 
-            #ip link add br20 type bridge && \
-            #ip link set br20 master blue addrgenmode none && \
-            #ip link set br20 addr aa:bb:cc:00:00:21 && \
-            #ip addr add 20.20.20.2/24 dev br20 && \
-            #ip link add vni20 type vxlan local 10.0.0.2 dstport 4789 id 20 nolearning && \
-            #ip link set vni20 master br20 addrgenmode none && \
-            #ip link set vni20 type bridge_slave neigh_suppress on learning off && \
-            #ip link set vni20 up && \
-            #ip link set br20 up && \
-
+            ip link add br20 type bridge && \
+            ip link set br20 master blue addrgenmode none && \
+            ip link set br20 addr aa:bb:cc:00:00:21 && \
+            ip addr add 20.20.20.2/24 dev br20 && \
```

Comment:
Why have you added an IP address here and not in the br100 configuration block ?

Reply:
We should not have an IP on VLAN100 as it is used for L3VNI.